src/App.tsx

```diff
@@ -1,156 +1,84 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterBy } from './types/FilterBy';
+import { ErrorMessage } from './types/ErrorMessage';
+import { filterByStatus } from './utils/helpers';
+
+const USER_ID = 10913;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorNotification, setErrorNotification] = useState(ErrorMessage.none);
```

Comment:
It would be more organic to have `ErrorMessage | null` instead of an empty `ErrorMessage` value